.golangci.yml

```
@@ -0,0 +1,35 @@
+service:
+  golangci-lint-version: 1.21.0
```

Let's just update now
```suggestion
  golangci-lint-version: 1.23.0
```